#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *;
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *;

from math import pi

gen = ParameterGenerator ()

gen.add("top_n_ratio", double_t, 0, "", 0.9, 0.0, 1.0)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "ExtractParticlesTopNBase"))
